package.json

```diff
@@ -151,7 +151,7 @@
     "electron-notarize": "^1.0.0",
     "electron-reloader": "^1.2.1",
     "eslint": "^7.6.0",
-    "eslint-config-expensify": "2.0.23",
+    "eslint-config-expensify": "^2.0.24",
```

Comment:
Should we prevent using `^` here? I think if we add new lint rule then it will fail for others if they do fresh npm i, unless all errors are fixed before.
Just my thought. What do you think?

Reply:
Oh yeah good point.